src/editor/widgets/select/SelectWidget.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Select from 'react-select';
+import Label from '../../Label';
+
+const CLASSIFYING = 'classifying'
+
+const baseSelectOption = args => ({
+  type: 'OptionSelection', value: '', purpose: CLASSIFYING, ...args
+})
+
+const SelectWidget = props => {
+  // Use the context to determine which option to delete in the annotation.bodies
+  // since we're using multiple SelectWidgets
+  const {disabled, annotation, config} = props
+  const {options, context} = config
+  let {placeholder} = config
+
+  const selectedOption = !annotation ? null : 
+    annotation.bodies.find(b => b.purpose === CLASSIFYING && b.context === context);
+
+  const onChange = option => {
+    if (!option && selectedOption) {
+      props.onRemoveBody(selectedOption)
+    } else if (option) {
+      const {value} = option
+      const updatedOption = baseSelectOption({value, context});
+      if (selectedOption) {
+        props.onUpdateBody(selectedOption, updatedOption)
+      } else {
+        props.onAppendBody(updatedOption);
+      }
+    }
+  }
+
+  let selectedOptionValue = null;
+  if (selectedOption) {
+    selectedOptionValue = options.find(({value}) => value === selectedOption.value)
+  }
+  // Change the placeholder if there isn't an option and it's disabled
+  if (!selectedOptionValue && disabled) {
+    placeholder = 'None'
+  }
+  return (
+      <Select
+        value={selectedOptionValue}
+        onChange={onChange}
+        options={options}
+        placeholder={placeholder}
+        isDisabled={disabled}
+        isClearable
+      />
```

Comment:
Yes, we'll be handling the styles of all widgets in a separate ticket. For now, we just get the functionality figured out since changing styles is trivial.